platform.md

```diff
@@ -1003,6 +1007,9 @@ Where:
 [run-image]
```

Comment:
```suggestion
stack-id = "<stack ID>"
os = "<operating system>"
arch = "<architecture>"

[run-image]
```

`stack-id` may be unnecessary here because we require `CNB_STACK_ID` to be set, but it makes sense to be consistent I think.

Reply:
If these don't match the values in `analyzed.toml`, fail.

Reply:
Addressed in 0b5a6e0 of #172

I think introducing the `stack-id` here allows us to remove it as input from the `exporter`, which can read it from this file.